src/types/entity/File.d.ts

```diff
@@ -0,0 +1,14 @@
+import Entity from './Entity';
+
+declare class File extends Entity {
+    setName(name: string): void;
+    getName(): string | null;
```

Comment:
do we really want to allow having null or there should be a constructor? Is there plan to change it in future or it would say as is. Asking, because File doesn't look like should be with no name, no content, no size and type.

Reply:
Unfortunately, some of the legacy APIs still return null for the file name